root/layout/components/BottomMenu.js

```diff
@@ -202,10 +202,10 @@ const SearchMenu = () => (
     </span>
     <ul>
       <li>
-        <a href="/search">{l('Search Entities')}</a>
+        <a href="/search">{l('Advanced Search (entities)')}</a>
       </li>
       <li>
-        <a href="/search/edits">{l('Search Edits')}</a>
+        <a href="/search/edits">{l('Advanced Search (edits)')}</a>
```

Comment:
I agree with Search Entities -> Advanced Search, but I'm not sure this is an improvement.

Reply:
Hmm I am also conflicted here, but this felt like we were losing the 'entities' distinction:
- Advanced Search
- Search Tags

Or was that superfluous anyway?

Reply:
I think it makes sense to assume that the default search is for the main database stuff, and that "Search edits" searchs edits :)

Reply:
Also it would be nice to use the same kind of phrasing among the entries of this menu.
- Either verbal:
  - Search Entities (Advanced)
  - Search Edits
  - Browse Tags
  - Browse Top CD Stubs
- Or nominal:
  - Advanced entity search
  - Edit search
  - Tags cloud
  - Top CD stubs

Reply:
Excellent point Yvanzo. I'm finding it difficult to pick between the two...
I am really trying to get the line 'Advanced search' in there, as that's what's causing issues for users.

So probably nominal:
- Advanced search
- Edit search
- Tag cloud
- Top CD Stubs

If Akshat is putting an advanced search button next to the regular search in the top bar it'll likely take care of the problem, but this is something we can tweak to help now

Reply:
That last proposal seems good :)